kratos/utilities/normal_calculation_utils.cpp

```diff
@@ -37,8 +296,7 @@ void NormalCalculationUtils::CalculateOnSimplex(
                 CalculateNormal2D(it,An);
         }
     } else if(Dimension == 3) {
-        array_1d<double,3> v1;
-        array_1d<double,3> v2;
+        array_1d<double,3> v1, v2;
         for(ConditionsArrayType::iterator it =  rConditions.begin(); it !=rConditions.end(); it++) {
             // Calculate the normal on the given condition
             if (it->GetGeometry().PointsNumber() == 3)
```

Comment:
what if CalculateOnSimplex is called on quads?

Reply:
Only triangles and tets are simplex. This specific method is maintained for performance reasons.

Reply:
yes, so this could be an error message. Just to avoid the mistake. In any case, this is fine with me.